pages/use-solid.md

```diff
@@ -34,6 +34,7 @@ You can pick a Provider in the following list. However, keep in mind that Solid
 | Provider | Responsible for Domain Name and Terms | Responsible for Hosting | Location of Hosting |
 |-----------------------------------|:---------------------------------------------------:|:-------------------------------------------------:|:-------------------:|:--------------------:|
 | [inrupt.net](https://inrupt.net) | [Inrupt, Inc.](https://inrupt.com/terms-of-service) | [Amazon](https://aws.amazon.com) | USA |
+| https://solid.community/|  |  | UK|
```

Comment:
Looks like `.community` domains aren't automatically parsed into links. We could also add the link to the support forum as "people behind it":

```suggestion
| [solid.community](https://solid.community/)| [solid.community GitLab group](https://gitlab.com/solid.community/support) |  | UK|
```

Also, are we sure it's hosted in the UK? (And if so, don't we know who's doing the hosting?)

Reply:
[administrators]( https://github.com/solid/process/blob/master/administrators.md) @jaxoncreed @justinwb @timbl would be the ones who could confirm that